assets/css/styles.css

```diff
@@ -0,0 +1,85 @@
+body{
```

Comment:
have a look at converting the positoning into one big flex box, this will make the design responsive and less work will be required to convert the website to a viable product for our users